andela/wg-rogue-one
# #157731617: Relation between WorkoutLog and WorkoutSession

### What does this PR do?
Creates a relation between the workout log and workout session
#### Description of Task to be completed?
Previously, wger only linked the workout session and workout log based on date and this presented some problems in certain areas since it was not able to follow foreign keys with the ORM is some location.

This PR creates a foreign key of the Workout Log.

#### How should this be manually tested?
1. Follow the readme.rst procedures on how to set up the application locally 
2. Make sure you run the migrations command
3. The database should contain a field for log_id in the workout session

#157731617
